docs/changes.txt

```diff
@@ -159,6 +159,8 @@ All (GUI):
 - Ensure that navigation order reflects layout of wxStdDialogButtonSizer.
 - Add Scintilla FineTicker methods to wxSTC (NewPagodi).
 - Add wxFontPickerCtrl::SetMinPointSize() (Andreas Falkenhahn).
+- Add Set/GetFooterText() and Set/GetFooterIcon() to class
```

Comment:
Could we fit this on a single line somehow? E.g. "Add Set/GetFooterText/Icon() to wxRichMessageDialog (Tobias Taschner)" would do it. Or maybe even just "Add footer support to ...".

Also, extremely important, I know, but please put a full stop at the end of the line to keep things consistent. TIA!